src/views/ItemView.js

```diff
@@ -815,10 +815,10 @@ class ItemView extends React.Component {
 
               <Row center="xs">
                 <Col sm={6}>
-                  <FormattedMessage
-                    id="ui-inventory.instanceTitle"
-                    values={{ title: instance.title }}
-                  />
+                  <FormattedMessage id="ui-inventory.instanceTitle" />
+                  <Link to={`/inventory/view/${instance.id}`}>
+                    {` ${instance.title}. `}
```

Comment:
@Baroquem I'm curious was there a reason to move the actual title out of `FormattedMessage`?

Reply:
I suppose it would be better not to, wouldn't it? Maybe something like this instead?
`    const linkedInstanceTitle = (
      <Link to={`/inventory/view/${instance.id}`}>
        {` ${instance.title}. `}
      </Link>
    );
`
`                <FormattedMessage
                    id="ui-inventory.instanceTitle"
                    values={{ title: linkedInstanceTitle }}
                  />
`